src/app/(sidebar)/smart-contracts/styles.scss

```diff
@@ -89,29 +89,51 @@
   }
 
   &[data-type="sym"] {
-    --DataTypeLegend-color: var(--sds-clr-navy-11);
+    --DataTypeLegend-color: var(--sds-clr-navy-10);
   }
 
-  &[data-type="string"] {
-    --DataTypeLegend-color: var(--sds-clr-gray-12);
+  &[data-type="u8-u64"] {
+    --DataTypeLegend-color: var(--sds-clr-teal-10);
   }
 
-  &[data-type="address"] {
-    --DataTypeLegend-color: var(--sds-clr-gray-11);
+  &[data-type="i8-i64"] {
+    --DataTypeLegend-color: var(--sds-clr-teal-08);
   }
 
-  &[data-type="i128-i256"],
-  &[data-type="u128-u256"]{
-    --DataTypeLegend-color: var(--sds-clr-lime-11);
+  &[data-type="i128-i256"] {
+    --DataTypeLegend-color: #2EBDE5;
   }
 
-  &[data-type="i8-i64"],
-  &[data-type="u8-u64"]{
-    --DataTypeLegend-color: var(--sds-clr-teal-11);
+  &[data-type="u128-u256"] {
+    --DataTypeLegend-color: #5FD4F4;
   }
 
   &[data-type="bool"] {
-    --DataTypeLegend-color: var(--sds-clr-pink-11);
+    --DataTypeLegend-color: var(--sds-clr-pink-10);
+  }
+
+  &[data-type="map"] {
+    --DataTypeLegend-color: #EE9D2B;
+  }
+
+  &[data-type="vec"] {
+    --DataTypeLegend-color: #5BB98C;
+  }
+
+  &[data-type="string"] {
+    --DataTypeLegend-color: #687076;
+  }
+
+  &[data-type="address"] {
+    --DataTypeLegend-color: #7E868C;
+  }
+
+ &[data-type="bytes"] {
+    --DataTypeLegend-color: #889096;
+  }
+
+  &[data-type="bytesN"] {
+    --DataTypeLegend-color: #C1C8CD;
   }
```

Comment:
We should never use hard-coded colors. If there are no color vars in the SDS, let's pick the closest one from what we have.